config/stage.json

```diff
@@ -72,6 +72,7 @@
 		"post-editor/html-toolbar": true,
 		"post-editor/insert-menu": true,
 		"post-editor/image-editor": true,
+		"post-editor/mentions": true,
```

Comment:
I'm curious...would we ever deploy a new feature directly into Production? What's the criteria for deciding this?